dashboard/test/ui/features/plc/pd/dashboard_view.feature

```diff
@@ -3,13 +3,14 @@
 
 Feature: Basic appearance for Facilitator Survey UI
 
-@skip
-Scenario: Facilitator View of surveys is as expected
-  Given I am a facilitator with completed courses
-  And I am on "http://studio.code.org/pd/workshop_dashboard/survey_results"
-  And I wait to see "#surveyResultsTable"
-  And I open my eyes to test "facilitator survey results"
-  And I see no difference for "viewing facilitator survey results"
+Scenario: Facilitator View of dashboard is as expected
+  Given I am a facilitator with started and completed courses
+  And I am on "http://studio.code.org/pd/workshop_dashboard"
+  And I wait to see element with ID "endedWorkshopsTable"
+  And I wait to see element with ID "inProgressWorkshopsTable"
+  And I wait to see element with ID "notStartedWorkshopsTable"
+  And I open my eyes to test "facilitator dashboard"
+  And I see no difference for "viewing facilitator dashboard"
```

Comment:
The entire test changes here because we are reversing an error in https://github.com/code-dot-org/code-dot-org/pull/35749 when the test was disabled.  This actually puts the test back the way it originally was.

Reply:
That was me (bad copy/paste), thanks for fixing!